docs/sources/developers/plugins/data-frames.md

```diff
@@ -6,15 +6,13 @@ title: Data frames
 
 Grafana supports a variety of different data sources, each with its own data model. To make this possible, Grafana consolidates the query results from each of these data sources into one unified data structure called a _data frame_.
 
-The data frame structure is a concept that's borrowed from data analysis tools like the [R programming language](https://www.r-project.org), and [Pandas](https://pandas.pydata.org/).
-
-> Data frames are available in Grafana 7.0+, and replaced the Time series and Table structures with a more generic data structure that can support a wider range of data types.
-
-This document gives an overview of the data frame structure, and of how data is handled within Grafana.
+> **Note:** Data frames are available in Grafana 7.0+, and replaced the Time series and Table structures with a more generic data structure that supports a wider range of data types.
 
 ## The data frame
 
-A data frame is a columnar-oriented table structure, which means it stores data by column and not by row. To understand what this means, let’s look at the TypeScript definition used by Grafana:
+Data frames have a columnar-oriented table structure, which means it stores data by column and not by row. 
+
+You can see what this means by looking at the TypeScript definition used by Grafana:
 
 ```ts
 interface DataFrame {
```

Comment:
Is this out of date? https://github.com/grafana/grafana/blob/c308118fc024eae0a7bf48a0b681e234e37262c3/packages/grafana-data/src/types/dataFrame.ts#L236

Reply:
Good catch, I've commited a fix.